vignettes/articles/cookbook_advanced_use_cases.Rmd

```diff
@@ -417,54 +417,157 @@ Suppose a bank aims to reduce transition risk in its loan book to align with its
     metric %in% c("projected", "target_sds"),
-    year == 2025,
+    year %in% c(2020, 2025),
     technology == "coalcap"
   ) %>% 
-  select(name_abcd, sector, technology, metric, year, technology_share) %>%
-  mutate(technology_share = round(technology_share, 3)) %>% 
-  pivot_wider(names_from = "metric", values_from = "technology_share") %>% 
-  mutate(deviation = round((projected - target_sds) / target_sds, 3))
+  select(name_abcd, sector, technology, year, metric, technology_share, production) %>%
+  mutate(
+    technology_share = round(technology_share * 100, 1),
+    production = round(production * 100)
+  )
+
+most_misaligned_coal_tech_mix <- misaligned_coal_tech_mix %>%
+  filter(year == 2025) %>% 
+  mutate(
+    deviation_tech_share = (technology_share - lead(technology_share)) / lead(technology_share) * 100,
+    .by = "name_abcd"
+  ) %>% 
+  arrange(desc(deviation_tech_share)) %>% 
+  head(3) %>% 
+  pull(name_abcd)
+  
+```
+
+The bank determines the three most misaligned coal power (based on future technology mix) and cement (based on future emission intensity) companies are the following, each of which it intends to engage with directly.
+
+```{r use_case_three_risk_mangement_result_power, echo = FALSE}
+# misaligned_coal_tech_mix
+misaligned_coal_tech_mix %>% 
+  filter(name_abcd %in% most_misaligned_coal_tech_mix) %>%
+result_table(
+  title = "Most misaligned coal power companies",
+  subtitle = "based on coal share in technology mix in 2025"
+) %>%
+  gt::cols_width(
+    name_abcd ~ px(110),
+    sector ~ px(80),
+    technology ~ px(100),
+    year ~ px(70),
+    metric ~ px(100)
+  ) %>%
+  gt::cols_label(
+    name_abcd = "Company",
+    sector = "Sector",
+    technology = "Technology",
+    year = "Year",
+    metric = "Metric",
+    technology_share = "Tech Share (% Capacity)",
+    production = "Production (MW Capacity)"
+  )
+
+```
+
+The bank can now engage with these companies to understand the reasons for the deviation and to help them develop a transition plan to reduce their exposure to high-carbon activities. A few calculations that help with determining the right approach to engaging these companies are the following:
+
+(A) The bank can calculate the percent deviation of the projected technology share relative to the target technology share of a company for a technology in t5 as follows:
+
+$$tech share_{comp, tech}^{deviation}(t_{5}) = \frac{tech share_{comp, tech}^{projected}(t_{5}) - tech share_{comp, tech}^{target}(t_{5})}{tech share_{comp, tech}^{target}(t_{5})}$$
+This can be a good first indicator of misalignment a the company level. A related calculation would be to calculate the percent deviation of the projected production (or capacity) level relative to the target production level of a company for a technology in t5, again as follows:
+
+$$prod_{comp, tech}^{deviation}(t_{5}) = \frac{prod_{comp, tech}^{projected}(t_{5}) - prod_{comp, tech}^{target}(t_{5})}{prod_{comp, tech}^{target}(t_{5})}$$
```

Comment:
I would define all of these variables (e.g. before "(A)"), before using them in an equation. 
That will also allow you to be a bit more streamlined in the naming. For example, you could say:
- Let $p_{t(5)}^{\Delta}$ be the companies deviation in production from t(5) 
- etc.

Reply:
This will make the formulae a lot more elegant/ easier to read.

Reply:
Looks great!